tests/tests.rs

```diff
@@ -211,3 +211,36 @@ fn test_list() {
     fail::cfg("list", "return").unwrap();
     assert!(fail::list().contains(&("list".to_string(), "return".to_string())));
 }
+
+#[test]
+fn test_value_adjust() {
```

Comment:
How about:
```
    let f = || -> i32 {
        let mut var = 1;
        fail::failpoint!("adjust_var", |s| {
            var = s.parse().unwrap();
           var
        });
        var
    };
    fail::cfg("adjust_var", "return(2)").unwrap();
```?

Reply:
what if the return value is not a number

Reply:
It doesn't matter as the value can always be parsed by from string in some way.